uploader.html

```diff
@@ -178,96 +175,88 @@ <h4 class="modal-title">Upload <%= filetype %></h4>
       var self = {},
           _private = {};
 
-      // private IDs and Classes 
+      self.init = function (filetype) {
+        var mediaTypes = [
+          {'image': {'text': 'image', 'glyphicon': 'glyphicon glyphicon-picture'}, 'audio': {'text': 'audio', 'glyphicon': 'glyphicon glyphicon-volume-up'}, 'video': {'text': 'video', 'glyphicon': 'glyphicon glyphicon-facetime-video'}, 'video-inline': {'text': 'video-inline', 'glyphicon': 'glyphicon glyphicon-play'}, 'expanded-audio': {'text': 'expanded-audio', 'glyphicon': 'glyphicon glyphicon-volume-up'},
+          }
+        ];
+        var mediaType = mediaTypes[0][filetype];
 
-      // init
-      self.init = function () {
         $(document).ready(function() {
-          createModalHeader();
+          createModalHeader(mediaType['text']);
+        });
+        $(document).on('click', opts.selectFileSelector, function() {
+          _private.createSelectFileSection(mediaType['text']);
         });
-        $(document).on('click', opts.selectFileSelector, _private.createSelectFileSection);
         $(document).on('change', opts.fileFieldSelector, function(event) {
```

Comment:
Should be `$(opts.fileFieldSelector).change(function(event) {`

Reply:
Why is that? I believe Sarah's using `on` here because `$(opts.fileFieldSelector)` may not exist at the time this code runs.

Reply:
oh, that makes sense. I hadn't thought about it not existing.